src/types/drupal/node.ts

```diff
@@ -59,7 +59,7 @@ export type NodeTypes =
   | NodeEventListing
 
 /** Node resource types. */
-export const enum NodeResourceType {
+export enum NodeResourceType {
```

Comment:
this was a good catch, but now I'm wondering we need these enums at all (here and in paragraph), or if they just provide confusion. Considering we have basically equivalent consts up in `src/lib/constants/resourceTypes.ts` that we actually make use of

Reply:
This is precisely what I want to take on next. A second round of type cleanup. This slowed me down a _lot_, but I just decided that I needed to get _something_ working and then come back and clean up as a separate ticket.

I'm not sure which I like better - the `enum` vs. the key/val JS object. There are advantages and disadvantages to both.